src/main/java/org/snakeyaml/engine/v2/constructor/ConstructScalar.kt

```diff
@@ -31,7 +31,7 @@ abstract class ConstructScalar : ConstructNode {
 
     companion object {
         @JvmField
-        protected val BOOL_VALUES: Map<String, Boolean> = mapOf(
+        internal val BOOL_VALUES: Map<String, Boolean> = mapOf(
```

Comment:
Kotlin doesn't allow `@JvmField` + `protected` for some obscure reason - it causes a compiler error.